packages/financial-templates-lib/src/clients/InsuredBridgeL2Client.ts

```diff
@@ -47,6 +47,10 @@ export class InsuredBridgeL2Client {
     return Object.keys(this.deposits).map((depositHash: string) => this.deposits[depositHash]);
   }
 
+  getAllDepositsForL1Token(l1TokenAddress: string) {
```

Comment:
this method enables the fix for the bug. see https://github.com/UMAprotocol/protocol/pull/3544/files#r740999431 to see how it's used.